core/src/impl/Kokkos_Profiling.hpp

```diff
@@ -59,7 +59,31 @@ namespace Kokkos {
 bool tune_internals() noexcept;
 
 namespace Tools {
-
+namespace Impl {
+template <class T>
+struct TuningResult;
+template <class Policy>
+struct TuningResult {
+  using type = Policy;
+};
+template <class input>
+struct translate {
+  using type = input;
+};
+template <>
+struct translate<Kokkos::Experimental::TuneOccupancy> {
+  using type = Kokkos::Experimental::DesiredOccupancy;
+};
+template <class... Properties>
+struct TuningResult<Kokkos::RangePolicy<Properties...>> {
+  // using type = Kokkos::RangePolicy<Properties...>;
+  using type = Kokkos::RangePolicy<typename translate<Properties>::type...>;
+};
```

Comment:
"The type of tuning a RangePolicy with the TuneOccupancy tag is a RangePolicy with the DesiredOccupancy tag"

Reply:
No longer needed